CHANGELOG.md

```diff
@@ -17,6 +17,8 @@ For details about compatibility between different releases, see the **Commitment
 
 ### Removed
 
+- Removed `attributes` from `GatewayAntenna` definition.
```

Comment:
You can move under `Changed` now, since we basically only deprecated it. You may also add a small explanation to this change and mention that the underlying store never stored the values here.